README.md

```diff
@@ -7,10 +7,13 @@ This is my humble attempt to code in Java...
 
 ### Algorithms
 
-
-- <a href="#">Example 2</a>: Convert a binary number to its decimal representation
-- <a href="#">Example 3</a>: Fibonacci Sequence
-- <a href="src/main/java/applications/algorithms/FindMinimumTwoSortedSubArrays.java">FindMinimumTwoSortedSubArrays</a>: Find the minimum element between two increasingly sorted sub-arrays
+- <a href="- <src/main/java/applications/algorithms/FibonacciSequence.java">Example 1</a>: Fibonacci Sequence
```

Comment:
I think this ```href``` is problematic...check how you did the rest...

Reply:
yep, fixed it. should be good by now